src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -194,14 +201,69 @@ export class IndexPattern implements IIndexPattern {
       };
     });
 
+    const runtimeFields = {
+      ...this.getComputedRuntimeFields(),
+      ...this.getComputedRuntimeObjects(),
+    };
+
     return {
       storedFields: ['*'],
       scriptFields,
       docvalueFields,
-      runtimeFields: this.runtimeFieldMap,
+      runtimeFields,
     };
   }
 
+  private getComputedRuntimeFields() {
```

Comment:
Those are the runtime fields (all but the 'object' subFields) that are sent in the search request